pkg/core/resources/model/resource.go

```diff
@@ -80,6 +80,8 @@ type ResourceSpec interface {
 }
 
 type ResourceList interface {
+	GetTotal() uint32
+	SetTotal(uint32)
```

Comment:
I think it makes sense to push it into Pagination like this:
```
type Pagination struct {
	NextOffset string
	Total uint32
}
```
since it's strictly connected with Pagination

As a user of `ResourceList` I would expect that `resourceList.GetTotal()` returns just number of elements in the list.

What do you think?

Reply:
Well, I am not sure that the `total` field in the response is directly related to pagination. It can be used there of course, but the primary target is to provide information about all available items in that resource. The GUI will use it to give an overview of the number of resource that particular control plane manages.
So for me, this is the proper place to handle the Total internals in the ResourceList interface,

Reply:
OK, I moved it to Pagination as suggested. This makes the code cleaner. Thanks!